### PR TITLE
fix: current group view in header

### PR DESCRIPTION
**Pull request by antonpanchenk0**

### CODE REVIEW

**Review by staspitsyk — APPROVED** [review]

File: front/src/components/group/groupHeader/index.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ class GroupHeader extends Component {
         avatar: this.props.currentGroup.avatar,
     }
 
+
+    componentDidUpdate(prevProps, prevState, snapshot) {
+        if(JSON.stringify(prevProps.currentGroup) !== JSON.stringify(this.props.currentGroup)) {
```
Comment (Alan18081) [inline front/src/components/group/groupHeader/index.js]: It is better to compare ids of groups

File: front/src/components/group/groupHeader/index.js [hunk 2]
```diff
@@ -17,6 +17,17 @@ class GroupHeader extends Component {
         avatar: this.props.currentGroup.avatar,
     }
 
+
+    componentDidUpdate(prevProps, prevState, snapshot) {
+        if(prevProps.currentGroup.id !== this.props.currentGroup.id) {
```
Comment (Alan18081) [inline front/src/components/group/groupHeader/index.js]: Please, add check that `prevProps.currentGroup` and `this.props.currentGroup`are not `undefined`

File: front/src/components/group/groupHeader/index.js (hunk 1 above)
Comment (antonpanchenk0) [reply front/src/components/group/groupHeader/index.js]: done

File: front/src/components/group/groupHeader/index.js (hunk 2 above)
Comment (antonpanchenk0) [reply front/src/components/group/groupHeader/index.js]: done

**Pull request merged**